src/components/StripCard/StripCard.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import styles from './StripCard.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+
+export interface StripCardProps {
+  media?: React.ReactNode;
+  info?: React.ReactNode;
+  roundMedia?: boolean;
+  withoutSidePadding?: boolean;
+}
+
+interface DefaultProps {
+  roundMedia: boolean;
+  withoutSidePadding: boolean;
+}
+
+export class StripCard extends React.Component<StripCardProps> {
+  static displayName = 'StripCard';
+  static defaultProps: DefaultProps = {
+    roundMedia: false,
+    withoutSidePadding: false,
```

Comment:
> I think it's better to avoid double negatives, and have this prop be sidePadding, and have a default value of true. I believe It's a bit more intuitive.

`sidePadding: true`